src/core/StelTextureMgr.cpp

```diff
@@ -17,12 +17,12 @@
  * Foundation, Inc., 51 Franklin Street, Suite 500, Boston, MA  02110-1335, USA.
  */
 
-#include "StelApp.hpp"
+//#include "StelApp.hpp"
```

Comment:
Why comment out instead of removing?

Reply:
Also these are suggested for removal (or change) by clangd ("not directly used" or so). 
I first commented out to test on other systems (or CI, as it seems some systems react differently, or in case I don't have all plugins active). These will be removed before merge.